Update Bootstrap and theme variables

dennisreimann opened the pull request:
WIP until @dstrukt and I refine the sizing variables.

dstrukt reviewed (approved):
All looks good, already sent DMs about specific visual inconsistencies that have arrived, might post them here for good measure, but nice work!

dennisreimann commented on BTCPayServer/wwwroot/main/site.css:
```diff
@@ -8,12 +8,6 @@ h1, h2, h3, h4, h5, h6, .h1, .h2, .h3, .h4, .h5, .h6 {
-    width: calc(100% + 1rem);
-    margin: 1.5rem -.5rem;
-}*/
-
```
This pulled the table out of the grid by the width of the horizontal padding. The goal was to align the table content with the rest of the content -- now this isn't the case anymore:

![alignment](https://user-images.githubusercontent.com/886/131402505-d2182a0b-af63-4f15-aa38-f14e532c4684.png)

Also the vertical margins are missing without this. Was this an intended update?

dstrukt replied:
ahh, must have been removed when i was commenting, you can reverse (will comment this)

dennisreimann replied:
Thanks, re-added in dac4de1

dennisreimann commented:
@dstrukt I addressed the points you sent via Mattermost. Can you confirm I did what you proposed? If so, let's get this one merged and iterate further in another PR.

dstrukt commented:
Can confirm, and added a few additional changes (for review) to address the issues my requested changes created!

dennisreimann commented:
Let's get this merged, will rebase #2794 afterwards – that one is also basically done.

pavlenex commented:
It's a great update but some things are a bit weird. Like we've started to bold text and use way too much light green on the warnings box that it hurts my eyes.

![Screenshot 2021-09-01 at 14 19 52](https://user-images.githubusercontent.com/36959754/131670325-784baac8-0b80-4141-9bfd-a764986a57fb.png)
![Screenshot 2021-09-01 at 14 20 13](https://user-images.githubusercontent.com/36959754/131670329-d66329e8-483a-45ae-9de7-bc4058f4ce36.png)


![Screenshot 2021-09-01 at 14 17 31](https://user-images.githubusercontent.com/36959754/131670240-15281ec4-58d4-4ecf-9dfc-5d1ca2b6f6c1.png)
![Screenshot 2021-09-01 at 14 17 44](https://user-images.githubusercontent.com/36959754/131670249-8441bc55-2e56-4e6f-a094-b30a85349230.png)

Also, what's the reasoning behind expanding menu size?

![Screenshot 2021-09-01 at 14 23 24](https://user-images.githubusercontent.com/36959754/131670529-c6b76d55-a5a6-4051-8c3b-64fa2fb3b69e.png)
![Screenshot 2021-09-01 at 14 23 37](https://user-images.githubusercontent.com/36959754/131670534-3982d7d3-7990-450f-a225-de33438df312.png)


Personally I prefer light text, light menus and I am not really sure what kind of UX benefits we get with these changes?

I think this PR overdid certain things to the extand it made them look worse than before.

dennisreimann commented:
@pavlenex As discussed on Mattermost we reverted the sidebar font size change and also reset the font weight for links back to normal instead of semibold.

What else needs to be addressed so that we can move forward and get this merged?

pavlenex commented:
@dennisreimann This needs to be compared side by side with what we already have. I'll do that in a bit. 
Also there's a conflict that needs to be resolved.

dennisreimann commented:
Reduced the navbar padding for desktop view now (Reverted to how it was). The sidebar font being 16px might be a caching issue -- can you check out the latest commit, rebuild and then hard refresh the browser? (Shift Reload or clear the browser cache)

pavlenex reviewed (approved):
Had to re-clone the repo for some reason, but it works now. All good, thanks reverting.

tACK
